angular/projects/spark-angular/src/lib/components/sprk-tabs/sprk-tabs.component.ts

```diff
@@ -0,0 +1,365 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChildren,
+  ElementRef,
+  HostListener,
+  Input,
+  QueryList,
+  Renderer2,
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+import { SprkTabsPanelDirective } from '../../directives/sprk-tabs/sprk-tabs-panel/sprk-tabs-panel.directive';
+import { SprkTabsButtonDirective } from '../../directives/sprk-tabs/sprk-tabs-button/sprk-tabs-button.directive';
+
+@Component({
+  selector: 'sprk-tabs',
+  template: `
+    <div
+      [ngClass]="getClasses()"
+      [attr.data-id]="idString"
+      [attr.data-analytics]="analyticsString"
+    >
+      <div class="sprk-c-Tabs__buttons" role="tablist">
```

Comment:
Should we add an input to allow for additionalClasses on this element?

Reply:
I think that's a good idea